web/server/vue-cli/src/assets/userguide/userguide.md

```diff
@@ -14,429 +14,759 @@ on GitHub</a>
+Editing a product is done through the pencil icon which can be found in the
+latest *Actions* column of the product list table. This window lets you edit
+the product's configuration.
+
+![Edit product](images/products/edit_product.png)
 
-![Remove product](images/remove_product.png)
+### Remove a product
+Products can be deleted by clicking on the red trash bin in the latest
+*Actions* column of the product list table. This way the product is only
+unmounted from the server (losing access control data and connection), but
+**no analysis results are deleted**.
 
-## <a name="userguide-managing-permissions"></a> Managing permissions
-* Server-wide permissions can be edited by clicking *Edit global permissions*.
-* Product-level permissions can be edited by clicking the edit icon for the
-product you want to configure the permissions.
+![Remove product](images/products/remove_product.png)
+
+## Managing permissions
+* Server-wide permissions can be edited by clicking *EDIT GLOBAL PERMISSION*
+button above the product list page. Only users with `Super Users` access can
+edit global permissions.
+* Product-level permissions can be edited by clicking the edit icon
+(blue pencil) for the product you want to configure the permissions.
 
 The two lists show the users and groups known to the system - if a tick is
 present in its row, the given user or group has the permission directly
 granted. (Users who only have a certain permission through permission
 inheritance are not shown with a tick.)
 
-![Product permissions](images/product_permissions.png)
+![Product permissions](images/products/product_permissions.png)
 
 Only the permissions you have rights to manage are shown.
 
 You can edit multiple permissions opening the window only once. Simply tick or
 untick the users/groups you want to give the permission to or revoke from them.
 Clicking *OK* will save the changes to the database.
 
-# <a name="userguide-list-of-runs"></a> List of runs
-List page contains the analysis runs available on the server under the selected
-product.
+## Notification banner
+There is an option to set a global notification text, that will show on top of
+every page below the header bar.
+
+![Notification](images/products/notification.png)
+
+You can edit this alert on the products page with `EDIT ANNOUNCEMENT` button,
+but **Super User** permissions are necessary.
+
+![Products](images/products/edit_announcement_btn.png)
+
+You can remove the current notification by saving an empty text.
+
+![Notification Dialog](images/products/notificaiton_dialog.png)
 
-You can do the following on this page:
-- [Filter runs](#userguide-filtering).
-- [Compare analysis](#userguide-compare-runs).
-- [Delete runs](#userguide-delete-runs).
-- [Sorting runs](#userguide-sorting-runs).
+# Runs and history events
+## Runs
+A run represents the same analysis activity for the same source code with the
+same analysis settings (CTU, non-CTU). We are storing analysis results to a
```

Comment:
instead of the CTU or non-CTU probably mentioning the checker configuration would be more understandable for the users (CTU is very clang static analyzer specific feature)